spec/bowling_spec.js

```diff
@@ -0,0 +1,64 @@
+describe('Bowling Game', function() {
+
+  var bowlingGame;
+
+  beforeEach(function() {
+    bowlingGame = new BowlingGame();
+  });
+
+  describe('can recieve a "roll"', function(){
+
+    //don't really know if I've stubbed all these correctly..
+    it('which returns a number', function(){
+      spyOn(Math, 'random').and.returnValue(0.2);
+      expect(bowlingGame.roll()).toEqual(2);
```

Comment:
Can use expect(bowlingGame.roll()) >= 0 && bowlingGame.roll()) < 11); to test if the number is always between 0 and 10. So you don't need to stub